src/stores/AppStore.js

```diff
@@ -880,7 +880,7 @@ export default types
 
         let node;
 
-        while ((node = children.shift())) {
+        while ((node = children.shift() && children)) {
```

Comment:
if `children` is undefined it will fail on `shift()`
and if it's some magic which works for some reasons, it's better to document it!